pkg/polaris/config.go

```diff
@@ -24,95 +24,85 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"net/url"
 	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/rubrikinc/rubrik-polaris-sdk-for-go/pkg/polaris/graphql"
 )
 
-// UserAccount holds an RSC local user account configuration. Note that RSC
-// user accounts with MFA enabled cannot be used.
+// UserAccount holds an RSC local user account configuration. Depending on how
+// the local user account is stored, the Name field might hold the RSC account
+// name.
+//
+// Note, RSC user accounts with MFA enabled cannot be used.
 type UserAccount struct {
-	// Polaris account name.
-	Name string
-
-	// Polaris account username.
-	Username string
-
-	// Polaris account password.
-	Password string
+	Name     string // User account name.
+	Username string // RSC account username.
+	Password string // RSC account password.
 
-	// Optional Polaris API endpoint. Useful for running the SDK against a test
-	// service. Defaults to https://{Name}.my.rubrik.com/api.
+	// Optional RSC API endpoint. Useful for running the SDK against a test
+	// service. When omitted, it defaults to https://{Name}.my.rubrik.com/api.
 	URL string
 
+	accountName string
+	accountFQDN string
+	apiURL      string
 	envOverride bool
+	tokenURL    string
 }
```

Comment:
nit: could we annotate these new struct values?

Reply:
We primarily document the public API, in addition to this the `ServiceAccount`/`UserAccount` struct has additional documentation for the public access functions for each of the fields.